src/Kraken.cpp

```diff
@@ -7,14 +7,14 @@
 
 #include <czmq.h>
 #include <g2log.hpp>
-
 #include "Kraken.h"
 
 
 /// Constructing the server/Kraken that is about to be connected/impaled by the client/Harpoon
 Kraken::Kraken():
 mLocation(""), 
-mQueueLength(10),
+mQueueLength(1),
+mMaxChunkSize(10485760), //10MB
```

Comment:
This just gives the user the ability to change it to something else. 10MB is fine, but if they want it to be 1K for example, they can do that as well. I used this in testing as well (set it to 1K to ensure that tons of packets being sent doesn't affect performance).